pages/lists/[listId]/activity.tsx

```diff
@@ -31,16 +31,23 @@ interface ContributorListPageProps {
   isOwner: boolean;
 }
 
+export type ServerFilterParams = {
```

Comment:
Naming (nit): No need to specify `Server` in the type name.

```suggestion
export type FilterParams = {
```

Also, `FilterParams` is used on the list activity page but also elsewhere. Consider moving the type elsewhere and import it into the list activity page.